demisto_sdk/__main__.py

```diff
@@ -1399,6 +1401,27 @@ def doc_review(**kwargs):
     sys.exit(1)
 
 
+# ====================== integration-diff ====================== #
+@main.command(name="integration-diff",
+              help='''Check differences between two integrations, one old and one new.''')
```

Comment:
be a bit more specific what is being checked in the command, what is the purpose for the command.